shell/common/gin_converters/download_converter.h

```diff
@@ -0,0 +1,26 @@
+/*** OpenFin ***/
+// Copyright 2021 OpenFin. All rights reserved.
+
+#ifndef ELECTRON_SHELL_COMMON_GIN_CONVERTERS_DOWNLOAD_CONVERTER_H_
```

Comment:
There's already a converter in `shell/browser/api/electron_api_download_item.cc` - we can just add this one to the same file given it's quite small.